### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by dreamora**
I would like to propose to start with these minimal bug and feature request templates to ease the creation of the corresponding types of issues in the future and to serve as a base for additional types of issues where we see corresponding added value.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Bug Report
+
+<!--
+  Please fill in each section completely. Thank you!
```
Comment (agitator) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe "Please fill in a reproduceable description, including preconditions."?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Suggestion
+
+<!--
+  Please fill in each section completely. Thank you!
```
Comment (agitator) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Maybe "Reason or motivation this proposal was created."?

Comment (dreamora) [reply .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
  Your reason or motivation to create this proposal.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dreamora) [reply .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
  Please fill in a reproducible description, including preconditions.
```

**Review by agitator — APPROVED** [review]
lgtm

**Pull request merged**